### DISCUSSION TITLE
Need help getting the emulated SB16 to work in Alpine Linux

### DISCUSSION THREAD

**Discussion by HunterBoy344** [copy/v86#1080] [Q&A] [answered]
By default, the snd-sb16 module doesn't even exist, just snd-sb-common and snd-sb16-dsp, so I can't modprobe it. I'm using Alpine Linux Standard with the linux-lts kernel. Installing linux-firmware-sb16 does not help. I can send my disk image if it is necessary.

I would really, really, really prefer not to have to recompile the entire Linux kernel. The disk image is only 3GB and I would like for it to stay that way. If I have to, though, I will, because I'm desperate at this point.

**Comment by SuperMaxusa** [comment, marked as answer]
Thank you for your issue on https://gitlab.alpinelinux.org/alpine/aports/-/issues/16296, now sound works on recent [linux-virt](https://pkgs.alpinelinux.org/package/edge/main/x86/linux-virt) in edge branch. 
You can replace your kernel on an installed system as follows:
```shell
$ apk add linux-virt --no-cache --repository https://dl-3.alpinelinux.org/alpine/edge/main
$ apk del linux-lts
# and restart.
```

https://github.com/user-attachments/assets/d06cc5a7-29ea-4721-bf3c-5e26d82db111

(sound may be a bit of buggy (like in my demo or https://github.com/copy/v86/issues/1049#issuecomment-2227033913), but that's other problem)

**Comment by HunterBoy344** [comment]
Excellent!

**Comment by HunterBoy344** [comment]
Ok, question: for me, the audio does work now in v86, but it's REALLY quiet. I've tried turning it up but it's still barely audible. What should I do?

**Comment by HunterBoy344** [comment]
Nvm, fixed it by changing some settings in alsamixer
